web/src/screens/_README.md

```diff
@@ -0,0 +1,10 @@
+# Screens
+
+## Why this folder?
+
+- Add complete screen components in this folder. Things like Dashboard, Profile
```

Comment:
Might go through and add "do not put screen components here, strictly widgets, etc." elsewhere